pkg/compute/rust.go

```diff
@@ -213,14 +218,8 @@ func (r Rust) Verify(out io.Writer) error {
 	if err != nil {
 		return fmt.Errorf("error fetching latest crate version: %w", err)
 	}
-	latestFastlySys, err := getLatestCrateVersion(r.client, "fastly-sys")
-	if err != nil {
-		return fmt.Errorf("error fetching latest crate version: %w", err)
-	}
 
-	// Create a semver constraint to be within the latest minor range or above.
-	// TODO(phamann): Update this to major when fastly-sys hits 1.x.x.
-	fastlySysConstraint, err := semver.NewConstraint(fmt.Sprintf("~%d.%d.0", latestFastlySys.Major(), latestFastlySys.Minor()))
+	fastlySysConstraint, err := semver.NewConstraint(fmt.Sprintf(">= %s <= %s", r.config.File.Language.Rust.FastlySysMin, r.config.File.Language.Rust.FastlySysMax))
```

Comment:
Can we just have this be a free-form constraint? If we ever have to yank a version of the crate we'd want this to be a disjoint range, not just a min-max.

Reply:
No problem. I've updated this 👍🏻

Reply:
Thank you! :tada: